modules/API/API.js

```diff
@@ -310,6 +314,29 @@ function initCommands() {
 
             APP.store.dispatch(sendTones(tones, duration, pause));
         },
+        'set-assumed-bandwidth-bps': value => {
+            logger.debug('Set assumed bandwidth bps command received', value);
+
+            if (isNaN(value)) {
```

Comment:
If you want to check if it's a number then use Number.parseInt. As it stands an object would pass this check for example. Or a string.

Reply:
Isn't it ok to pass? It's an early return. But I'll change it because of the empty string getting converted to 0 thing.

Reply:
By pass I mean it will be false so your code will continue as if it was a valid number. {} or [] are not NaN.

Reply:
Just {} is NaN, but indeed [] and "" would be problematic. Regardless, I'll change it again because Number.parseInt fails for 0.